src/styles/blocks/container.scss

```diff
@@ -0,0 +1,31 @@
+.container {
+  margin: 0 auto;

```

Comment:
Do not set margin for block, try to use mix.
![Screenshot 2022-08-02 at 12 48 30](https://user-images.githubusercontent.com/17098736/182345820-cdd7135f-0d95-495b-a03b-ad3f15806e45.png)